vault/seal/ocikms/ocikms.go

```diff
@@ -182,16 +180,9 @@ func (k *OCIKMSSeal) getOCIKMSClient() (*keymanagement.KmsCryptoClient, error) {
 
 // Request metadata includes retry policy
 func (k *OCIKMSSeal) getRequestMetadata() common.RequestMetadata {
-	if !KMSShouldRetryOperation {
-		noRetryPolicy := common.NoRetryPolicy()
-		return common.RequestMetadata{
-			RetryPolicy: &noRetryPolicy,
-		}
-	}
-
 	// Only retry for 5xx errors
 	retryOn5xxFunc := func(r common.OCIOperationResponse) bool {
-		return !(r.Error == nil && r.Response.HTTPResponse().StatusCode < 500)
+		return r.Error != nil || r.Response.HTTPResponse().StatusCode >= 500
```

Comment:
Should be `&&`, right?  We have an error, and it's code is a 5xx.

Reply:
Yes, I thought only 5xx has r.Error != nill, double checked and && is correct